requests_lb/requests_lb.py

```diff
@@ -135,34 +136,34 @@ def _retry_request(self, fn, target, **kw):
         Send the given request, as defined by the `fn` argument to the given
         target.
         """
+        _max_request_retries = self._max_request_retries
 
         if target[0] == '/':
             target = target[1:]
 
-        start = self._time()
-
-        while True:
+        while _max_request_retries > 0:
```

Comment:
When this condition becomes false there doesn't appear to be any follow through, and it seems like the method will return `None`. I expect there should be a `throw` at the bottom of this method saying something like `maximum retries attempted`.